root/programs/C#/Frameworks/Infrastructure/CustomControl/Control/WebCustomTextBox.cs

```diff
@@ -326,12 +326,12 @@ public bool Validate(out string[] result)
             if (this.CheckType != null)
             {
                 // 必須入力チェック
-                if (this.CheckType.IsIndispensabile)
+                if (this.CheckType.IsIndispensable)
                 {
                     if ((text == ""))
                     {
                         hasError = true;
-                        lstRet.Add(CmnCheckFunction.IsIndispensabileCheckErrorMessage);
+                        lstRet.Add(CmnCheckFunction.IsIndispensableCheckErrorMessage);
```

Comment:
@yusukemaegawa

I've registered this to the [issue](https://github.com/OpenTouryoProject/OpenTouryo/issues/149#issuecomment-178399677).